### PR TITLE
create_user template: modify displayed text

### PR DESCRIPTION
**Pull request by risicle**
Story https://trello.com/c/fNL5RPyB/81-update-the-create-account-page-to-remove-the-word-contributor-1

Now this looks like
new

### CODE REVIEW

**Comment by risicle** [comment]
Ok now it looks like this (action text changed as well as page "title" (not visible))
newer

File: app/templates/auth/create-user.html [hunk 1]
```diff
@@ -105,7 +102,7 @@ <h3 class="validation-masthead-heading" id="validation-masthead-heading">
             {%
               with
               type = "save",
-              label = "Create account" if role == 'buyer' else "Create contributor account"
+              label = "Create account" if role == 'buyer' else "Save and continue"
```
Comment (allait) [inline app/templates/auth/create-user.html]: Why "Save and continue" for suppliers? This the final step for account creation in both cases, so "Create account" should make sense in both cases. Or are there some new additional supplier steps after this one?

**Comment by risicle** [comment]
This was not my decision. See story on trello.

File: app/templates/auth/create-user.html (hunk 1 above)
Comment (risicle) [reply app/templates/auth/create-user.html]: This was not my decision. See story on trello.

Comment (risicle) [reply app/templates/auth/create-user.html]: @constancecerf

Comment (constancecerf) [reply app/templates/auth/create-user.html]: If you are going from the create a supplier journey, users think they've already created the account and now they are just creating a login to activate it. So the other option would be to say either 'Complete account' or 'Confirm account'

Comment (constancecerf) [reply app/templates/auth/create-user.html]: I've just had a chat with Karl, and @risicle, we can just be consistent with what's already in the code as long as it makes sense here.

Comment (allait) [reply app/templates/auth/create-user.html]: @constancecerf so can we keep "Create account" for both buyers and suppliers?

**Pull request merged**